### PR TITLE
Clarify airnode-examples README for users and devs

### PR DESCRIPTION
**Pull request by dcroote**
- Removes instructions pertaining to building docker images given we're now using released images
- Notes that released images are being used during deployment and local client flows
- Adds instructions for devs to use custom images (e.g. for testing PRs such as `api3/airnode-deployer-dev:bb9b8118940ec852c4223b13eba5a6eb97aa3b97` @kolenic-martin )

### CODE REVIEW

File: packages/airnode-examples/README.md [hunk 1]
```diff
@@ -175,55 +175,43 @@ Refer to the
...
-### 10. (Only if deploying to a cloud provider) Build deployer container
+Now you're ready to deploy Airnode on the cloud provider. To proceed, run:
 
 ```sh
-yarn rebuild-deployer-container
+yarn deploy-airnode
 ```
 
-This command will use the previously built artifacts container to build the deployer.
+This command will use the released deployer Docker image corresponding to the tag checked out during Setup. The image
```
Comment (Siegrift) [inline packages/airnode-examples/README.md:186]: Nit: Maybe link to "Setup" section?

**Review by Siegrift — COMMENTED** [review]
> Adds instructions for devs to use custom images (e.g. for testing PRs such as api3/airnode-deployer-dev:bb9b8118940ec852c4223b13eba5a6eb97aa3b97

Is this added somewhere? I do not see it in the README, but I agree it would be nice to add as it's currently only in our private slack message.

File: packages/airnode-examples/README.md (hunk 1 above)
Comment (dcroote) [reply packages/airnode-examples/README.md]: Added.

**Comment by dcroote** [comment]
Yes at the end of both new Steps 9 and 10 there is: "If you are a developer and would like to use a different..." I've updated the example commands used there to be `-dev` images to be clearer.

**Review by Siegrift — APPROVED** [review]
👍 LGTM

**Pull request merged**